cluster/gce/config-default.sh

```diff
@@ -319,28 +320,27 @@ fi
 #   new subnetwork will be created for the cluster.
 ENABLE_IP_ALIASES=${KUBE_GCE_ENABLE_IP_ALIASES:-false}
 NODE_IPAM_MODE=${KUBE_GCE_NODE_IPAM_MODE:-RangeAllocator}
-if [ ${ENABLE_IP_ALIASES} = true ]; then
+if [ "${ENABLE_IP_ALIASES}" = true ]; then
   # Number of Pods that can run on this node.
   MAX_PODS_PER_NODE=${MAX_PODS_PER_NODE:-110}
   # Size of ranges allocated to each node.
-  IP_ALIAS_SIZE="/$(get-alias-range-size ${MAX_PODS_PER_NODE})"
+  IP_ALIAS_SIZE="/$(get-alias-range-size "${MAX_PODS_PER_NODE}")"
+  export IP_ALIAS_SIZE
   IP_ALIAS_SUBNETWORK=${KUBE_GCE_IP_ALIAS_SUBNETWORK:-${INSTANCE_PREFIX}-subnet-default}
   # If we're using custom network, use the subnet we already create for it as the one for ip-alias.
   # Note that this means SUBNETWORK would override KUBE_GCE_IP_ALIAS_SUBNETWORK in case of custom network.
   if [[ "${CREATE_CUSTOM_NETWORK}" == true ]]; then
-    IP_ALIAS_SUBNETWORK="${SUBNETWORK}"
+    export IP_ALIAS_SUBNETWORK="${SUBNETWORK}"
   fi
-  # Reserve the services IP space to avoid being allocated for other GCP resources.
-  SERVICE_CLUSTER_IP_SUBNETWORK=${KUBE_GCE_SERVICE_CLUSTER_IP_SUBNETWORK:-${INSTANCE_PREFIX}-subnet-services}
```

Comment:
Currently `SERVICE_CLUSTER_IP_SUBNETWORK` is not used anywhere.
https://cs.k8s.io/?q=SERVICE_CLUSTER_IP_SUBNETWORK&i=nope&files=&repos=

It seems that util.sh uses the variable, but #51690 removed it from util.sh.
Therefore, I remove it.

Reply:
cc @MrHohn

Reply:
Thanks, SGTM.